icu4c/source/data/misc/supplementalData.txt

```diff
@@ -2104,7 +2104,7 @@ supplementalData:table(nofallback){
             "islamic-tbla",
         }
     }
-    cldrVersion{"37"}
+    cldrVersion{"38"}
```

Comment:
I don't know if this is wanted. You should be able to force the cldr version from the Ant build file.
There are lots of options there you might want to read up on.

Reply:
-DcldrVersion=37 cleaned this up, thanks!